Content.Shared/Dice/LoadedDiceComponent.cs

```diff
@@ -0,0 +1,57 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Dice;
```

Comment:
This probably shouldn't be in shared, since clients shouldn't know if a dice is loaded or not without holding it.

Reply:
I don't think it matters because it already has the loaded dice user interface on it which is in shared + the protoId already gives away that its loaded

Reply:
Prototype IDs probably shouldn't be sent to clients in release builds. But you're right about the ui, you would need to tell the client about it only when the dice is held. I don't know of a good way to do that.

Reply:
Related: I was considering (on @beck-thompson's suggestion) moving the `LoadedDiceSystem` to shared in order to avoid the noticeable delay when right-clicking before the special verb appears, but I think in the long run it makes more sense to keep as much of it in the server side and try to keep information secret from clients. That's obviously not fully possible until at least some sort of refactor of the bound UI system that allows for more "private" UI, but that seems like the better direction to aim towards.